### PR TITLE
Fixing basic typos in Doc Comments

### PR DESCRIPTION
**Pull request by Fourchaux**
Files checked : *.rs files inside 'src' directory.

Only Doc comments (/// and //!) are fixed.

Note: the non American-English spellings are left unchanged.

### CODE REVIEW

**Comment by Mark-Simulacrum** [comment]
Thanks!

@bors r+ rollup

File: src/libstd/panicking.rs [hunk 1]
```diff
@@ -221,9 +221,9 @@ impl<'a> PanicInfo<'a> {
     ///
     /// panic::set_hook(Box::new(|panic_info| {
     ///     if let Some(location) = panic_info.location() {
-    ///         println!("panic occured in file '{}' at line {}", location.file(), location.line());
```
Comment (frewsxcv) [inline src/libstd/panicking.rs]: there is now a tidy violation on this line, it needs to be wrapped

```
tidy check (x86_64-unknown-linux-gnu)
tidy error: /checkout/src/libstd/panicking.rs:224: line longer than 100 chars
```

File: src/libstd/panicking.rs [hunk 2]
```diff
@@ -221,7 +221,7 @@ impl<'a> PanicInfo<'a> {
     ///
     /// panic::set_hook(Box::new(|panic_info| {
     ///     if let Some(location) = panic_info.location() {
-    ///         println!("panic occurred in file '{}' at line {}", location.file(), location.line());
+    ///        println!("panic occurred in file '{}' at line {}", location.file(), location.line());
```
Comment (frewsxcv) [inline src/libstd/panicking.rs]: By convention, Rust uses four spaces for indentation, you'll need to do something like this instead:

```rust
    ///        println!("panic occurred in file '{}' at line {}", 
    ///                 location.file(), location.line());
````

Comment (Fourchaux) [inline src/libstd/panicking.rs]: Ah , ok. Thank you.

**Comment by frewsxcv** [comment]
Thanks!

@bors r=Mark-Simulacrum

File: src/libstd/panicking.rs (hunk 1 above)
Comment (frewsxcv) [inline src/libstd/panicking.rs]: Now there's a new tidy error:

```
tidy error: /checkout/src/libstd/panicking.rs:224: trailing whitespace
```

You're going to need to remove that space at the end of this line

**Comment by steveklabnik** [comment]
> Note: the non American-English spellings are left unchanged.

We specifically do use `en-US` for the standard library docs, so changing these would also be welcome!

**Review by steveklabnik — APPROVED** [review]

**Comment by steveklabnik** [comment]
Traivis is failing due to 

    [00:02:11] tidy error: /checkout/src/libstd/panicking.rs:224: trailing whitespace

After travis passes and the commits are squashed, r=me!

**Comment by aidanhs** [comment]
Hi @Fourchaux, have you had a chance to look at the trailing whitespace issue raised by the tidy check?

**Comment by aidanhs** [comment]
Oh and it'd be great if you could squash your commits into one while you're at it!

**Comment by aidanhs** [comment]
Hi @Fourchaux, as this has been inactive for a while I'm going to it to keep the PR queue tidy - if you do get round to updating, feel free to reopen. Thanks again for the changes!

**Pull request closed without merging**